src/components/App/App.js

```diff
@@ -45,9 +50,15 @@ function App() {
 	
   function fetchAllMovies() {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-    .then(response => response.json())
```

Comment:
changed the .then() to make sure the response was ok. If the response wasn't ok, we'd throw an error, with the value of `${response.status}: ${response.statusText}`.